setup.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python
+
+from distutils.core import setup
+from distutils.cmd import Command
+class TestCommand(Command):
+    user_options = []
+
+    def initialize_options(self):
+        pass
+
+    def finalize_options(self):
+        pass
+
+    def run(self):
+        import sys, subprocess
+
+        raise SystemExit(
+            subprocess.call(['nosetests',
```

Comment:
yes i am sure that the tests runner can be generated, I just did not know how. I think one problem is that it is missing .py in the code so some of the standard test generators dont work.